src/android_utils.py

```diff
@@ -24,9 +28,23 @@
 Uri = autoclass("android.net.Uri")
 
 ANDROID_VERSION = autoclass("android.os.Build$VERSION")
+RELEASE = ANDROID_VERSION.RELEASE
 SDK_INT = ANDROID_VERSION.SDK_INT
 
 
+# Path.is_relative_to only on python 3.9+.
+if not hasattr(Path, "is_relative_to"):
+
+    def _path_is_relative_to(self, *other):
+        try:
+            self.relative_to(*other)
+            return True
+        except ValueError:
+            return False
```

Comment:
Great, this is exactly the implementation in my 3.9: `print(inspect.getsource(Path.is_relative_to))`.